### ISSUE TITLE
Meteor Debugging Tools Support?

### ISSUE THREAD

**Issue by joevbruno** [juliancwirko/react-redux-webpack-meteor#7]
I cannot get the Meteor [devtools](https://github.com/bakery/meteor-devtools) to work this approach. Is this just a known trade-off or am I doing something wrong? I am not passing any additional configuration options.

**Comment by juliancwirko** [comment]
Hi, I don't use Meteor devtools, but I guess that you need to use it with real Meteor app and here you have separated front-end React/Redux app which is connected through DDP protocol with separated Meteor backend app, so I guess it could be the case.

**Comment by joevbruno** [comment]
Hey. Thanks for the quick response! I'm new to meteor. A client of mine is using it with a React front end. I've been writing React apps for the last 2.5 years - so really comfortable there (but I don't like the limitations / work arounds of doing React in a meteor app). Thus, your proposal here of separating the front and back ends of the app is really appealing to me. Is there any tooling you use or would recommend for meteor that will work with this approach? Do you just debug with console.logs or what?

**Comment by juliancwirko** [comment]
yes, I guess all debugging solutions used for Node/React apps will be applicable here. I don't use anything special.
